design/ceph-csi-operator.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Integrate Ceph-CSI operator with Rook
+target-version: release-1.15
+---
+
+# Integrate Ceph-CSI operator with Rook
+
+## Overview
+
+To integrate the new Ceph-CSI operator with Rook and remove the existing CSI deployment related code, RBAC's and dead code in later releases. Also before moving forward please read the ceph-CSI-operator design doc [here](https://github.com/ceph/ceph-csi-operator/blob/main/docs/design/operator.md).
+
+**Note**: This feature is experimental in v1.15 and will not come with multus support in v1.15 so, users who wish to use multus or already have multus based cluster should avoid using this feature.
```

Comment:
nit,

!!!note

Reply:
design docs are not rendered in official doc, so I skipped this